src/testing/jest/jest-27-and-under/jest-environment.ts

```diff
@@ -1,10 +1,13 @@
 import type { E2EProcessEnv, JestEnvironmentGlobal } from '@stencil/core/internal';
+import NodeEnvironment from 'jest-environment-node';
 
 import { connectBrowser, disconnectBrowser, newBrowserPage } from '../../puppeteer/puppeteer-browser';
+import { JestPuppeteerEnvironmentConstructor } from '../jest-apis';
 
-export function createJestPuppeteerEnvironment() {
-  const NodeEnvironment = require('jest-environment-node');
-  const JestEnvironment = class extends (NodeEnvironment as any) {
+export function createJestPuppeteerEnvironment(): JestPuppeteerEnvironmentConstructor {
+  const JestEnvironment = class extends NodeEnvironment {
+    // TODO(STENCIL-1023): Remove this @ts-ignore
```

Comment:
I am not considering this part of the "Jest Testing Improvements" initiative. This piece of tech debt has been around for some time. While I would like to get rid of it, it's not on the critical path (and like I said, it's _been_ here, adding stricter types just raised it)